syntax = "proto3";

package video_stream;

service VideoStream {
  rpc GetVideoList (VideoListRequest) returns (stream Video) {}
  rpc StreamVideo (VideoRequest) returns (VideoUrl) {}
}

message VideoListRequest {}

message Video {
  string name = 1;
}

message VideoRequest {
  string name = 1;
}

message VideoUrl {
  string url = 1;
}